packages/postcss-convert-values/src/lib/convert.js

```diff
@@ -31,41 +31,32 @@ function dropLeadingZero(number) {
   return value;
 }
 
-function transform(number, unit, conversion) {
-  const lowerCasedUnit = unit.toLowerCase();
```

Comment:
We already converted the unit to lower case to know whether to call this function, so we can pass the lower case unit directly instead of converting it once more.